test_scripts/API/SetAppIcon/006_Two_apps_are_registered_App1_iconResumed_true_App2_iconResumed_true.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0041-appicon-resumption.md
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) SDL, HMI are started.
+-- 2) App1 set custom icon via putfile and SetAppIcon requests.
+-- 3) App2 set custom icon via putfile and SetAppIcon requests.
+-- 4) Two app are re-registered.
+-- SDL does:
+-- 1) Registers an App 1 successfully, responds to RAI with result code "SUCCESS", "iconResumed" = true.
+-- 2) Registers an App 2 successfully, responds to RAI with result code "SUCCESS", "iconResumed" = true.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/SetAppIcon/commonIconResumed')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
+  syncFileName = "icon.png"
+}
+local requestUiParams = {
+  syncFileName = {
+    imageType = "DYNAMIC",
+    value = common.getPathToFileInStorage(requestParams.syncFileName)
+  }
+}
+local allParams = {
+  requestParams = requestParams,
+  requestUiParams = requestUiParams
+}
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+
+runner.Title("Test")
+runner.Step("App1 registration with iconResumed = false", common.registerAppWOPTU, { 1, false })
+runner.Step("Upload icon file", common.putFile)
+runner.Step("SetAppIcon", common.setAppIcon, { allParams } )
+
+runner.Step("App2 registration with iconResumed = false", common.registerAppWOPTU, { 2, false })
+runner.Step("Upload icon file", common.putFile)
```

Comment:
In such call of common.putFile will add file for app1 again, need to sends value for pAppID ->
runner.Step("Upload icon file", common.putFile, { nil, nil, 2 })